modules/internal/String.chpl

```diff
@@ -1643,22 +1689,38 @@ module String {
   //
   // Helper routines
   //
+  private const uint_A: uint(8) = ascii('A');
+  private const uint_Z: uint(8) = ascii('Z');
+  private const uint_a: uint(8) = ascii('a');
+  private const uint_z: uint(8) = ascii('z');
+  private const uint_0: uint(8) = ascii('0');
+  private const uint_9: uint(8) = ascii('9');
+  private const uint_space   : uint(8) = ascii(' ');
+  private const uint_tab     : uint(8) = ascii('\t');
+  private const uint_newline : uint(8) = ascii('\n');
+  private const uint_return  : uint(8) = ascii ('\r');
+
   private inline proc _byte_isUpper(b: uint(8)) : bool {
-    return b >= ascii('A') && b <= ascii('Z');
+    return b >= uint_A && b <= uint_Z;
   }
 
   private inline proc _byte_isLower(b: uint(8)) : bool {
-    return b >= ascii('a') && b <= ascii('z');
+    return b >= uint_a && b <= uint_z;
   }
 
   private inline proc _byte_isAlpha(b: uint(8)) : bool {
-    return b >= ascii('A')  && b <= ascii('z');
+    return b >= uint_A  && b <= uint_z;
   }
 
   private inline proc _byte_isDigit(b: uint(8)) : bool {
-    return b >= ascii('0')  && b <= ascii('9');
+    return b >= uint_0  && b <= uint_9;
   }
 
+  private inline proc _byte_isSpace(b: uint(8)) : bool {
```

Comment:
I'd also either get rid of the leading underscore (which should be OK since it's private?) or tack a 'chpl' on the front.  I'd like us to get away from leading underscores as much as possible going forward...  (in part because they're ugly-ish, in part because I'm not sure what I'm supposed to take from it...  unless we want to use it as a naming convention to mean private?  But I'm not sure I would want to...)